p2p/server.go

```diff
@@ -594,13 +594,13 @@ running:
 			// This channel is used by AddPeer to add to the
 			// ephemeral static peer list. Add it to the dialer,
 			// it will keep the node connected.
-			srv.log.Debug("Adding static node", "node", n)
+			srv.log.Trace("Adding static node", "node", n)
```

Comment:
I don't understand why those changes are needed.

Reply:
We are just downgrading the level of those particular logs, because they are quite spammy.

I understand that we might have different understanding of what `DEBUG` level is and what `TRACE` level is, and this probably falls in the category of style, so considering that I am not the author of this code, I would be happy to revert them if you prefer them to stay `DEBUG` level.